text/0000-semicolon-items.md

```diff
@@ -226,9 +214,8 @@ where `expr` is some expression and `label` is some label.
 # Reference-level explanation
 [reference-level-explanation]: #reference-level-explanation
 
-1. The token `;` is accepted as a valid item, both in the language syntax
-and by the `item` macro 'fragment specifier'. As an example, `struct Foo {};`
-is therefore in the language accepted by a Rust compiler.
+1. The token `;` is consumed and ignored between items in module, function, trait, and impl bodies. 
+   As an example, `struct Foo {};` is therefore in the language accepted by a Rust compiler.
 
 2. `rustfmt` will remove any extraneous `;` items.
```

Comment:
This still mentions `;` items.

I think this just just be removed actually since rustfmt doesn't remove extraneous `;` in statement context either.

Reply:
Hmm... shouldn't we go the other direction instead on statements?